src/inputs/input.css

```diff
@@ -0,0 +1,174 @@
+div.omrs-input-group {
+  margin-bottom: 1.5rem;
+  position: relative;
+  width: 20.4375rem;
+}
+
+input.omrs-input-underlined,
+input.omrs-input-filled,
+input.omrs-input-outlined {
+  font-size: 1rem;
+  display: block;
```

Comment:
Inputs are `display: inline` by default. Changing it to `display: block` will make inputs not stay on the same horizontal line like they normally do.

I would prefer not changing the default `display` for inputs, so that people used to how inputs behave don't have to learn/adjust to what OpenMRS styleguide is doing.

Are you using `block` so you can do vertical-align: middle? One way to solve that is to just add padding-top and padding-bottom to vertically center the text within an input.